store/sqlstore/user_access_token_store.go

```diff
@@ -149,20 +149,18 @@ func (s SqlUserAccessTokenStore) deleteTokensByUser(transaction *gorp.Transactio
 	return result
 }
 
-func (s SqlUserAccessTokenStore) Get(tokenId string) store.StoreChannel {
-	return store.Do(func(result *store.StoreResult) {
-		token := model.UserAccessToken{}
+func (s SqlUserAccessTokenStore) Get(tokenId string) (*model.UserAccessToken, *model.AppError) {
+	token := model.UserAccessToken{}
 
-		if err := s.GetReplica().SelectOne(&token, "SELECT * FROM UserAccessTokens WHERE Id = :Id", map[string]interface{}{"Id": tokenId}); err != nil {
-			if err == sql.ErrNoRows {
-				result.Err = model.NewAppError("SqlUserAccessTokenStore.Get", "store.sql_user_access_token.get.app_error", nil, err.Error(), http.StatusNotFound)
-			} else {
-				result.Err = model.NewAppError("SqlUserAccessTokenStore.Get", "store.sql_user_access_token.get.app_error", nil, err.Error(), http.StatusInternalServerError)
-			}
+	if err := s.GetReplica().SelectOne(&token, "SELECT * FROM UserAccessTokens WHERE Id = :Id", map[string]interface{}{"Id": tokenId}); err != nil {
+		if err == sql.ErrNoRows {
+			return nil, model.NewAppError("SqlUserAccessTokenStore.Get", "store.sql_user_access_token.get.app_error", nil, err.Error(), http.StatusNotFound)
+		} else {
```

Comment:
You don't need this else, you can just use the return it after the if because the if is always returning.

Reply:
Done with the change. :)